tests/microbenchmarks.py

```diff
@@ -167,21 +187,27 @@ def test_cmp(test_case):
                 x = [3] != 5.0
                 x = [3] > [5]
                 x = [3.0] > [5.0]
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_loops(test_case):
     print("Running", "test_loops")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils